R/FilterStateRange.R

```diff
@@ -515,6 +514,27 @@ RangeFilterState <- R6::R6Class( # nolint
         )
         invisible(NULL)
       })
+    },
+    ui_summary = function(id) {
+      ns <- NS(id)
+      uiOutput(ns("summary"), class = "filter-card-summary")
+    },
+    server_summary = function(id) {
```

Comment:
We should hide the summary when the filter card is clicked:
![image](https://user-images.githubusercontent.com/8597300/217100214-4b4890ce-e2ea-41b9-8cf0-8ffbd754dc4a.png)

Reply:
Will do.